client/app/index.js

```diff
@@ -2,6 +2,9 @@ import Vue from 'vue'
 import VueRouter from 'vue-router'
 import VuePolyglot from './plugins/vue-polyglot'
 
+import Dialog from './components/dialog'
+import Notification from './components/notification'
```

Comment:
I understand what you've got in mind with the `Notification` component, but isn't outside of the scope of this PR?